### ISSUE TITLE
Don't download cource

### ISSUE THREAD

**Issue by djzockre** [sajibcse68/linkedin-learning-downloader#1]
Don't download cource, close connection

**Comment by J3Patel** [comment]
@djzockre have a look at here, tested on July 2019
https://github.com/J3Patel/linkedin-learning-video-downloader

**Comment by sajibcse68** [comment]
@djzockre Sometimes it is needed to try multiple time since for some reason (maybe, login is not succeeded). Can you try again and let me know if it works?

**Comment by emon535** [comment]
Can't open the browser.

**Comment by sajibcse68** [comment]
The video should be downloaded using terminal. If you want to open browser then change `headless: true` to `headless: false` in index.js file

**Comment by xi1570-krupeshanadkat** [comment]
yep, not working. it is going to catch block with exception always.

**Comment by YoloAsh** [comment]
Launching the browser...
Login into Linkedin
Closing...
ERROR: The process with PID 23960 (child process of PID 100) could not be terminated.
Reason: The operation attempted is not supported.

PID number changes with every attempt

**Comment by YoloAsh** [comment]
i think this is probably because of my internet provider restrictions so my default browser can't open the page. I don't want to make tor as my default browser neither I want to install 3rd party vpn software. It seems I'm stuck here.
By any chance, will you add tor browser support? =)
